generate/genhcl/genhcl.go

```diff
@@ -174,9 +174,18 @@ func Load(rootdir string, sm stack.Metadata, globals stack.Globals) ([]HCL, erro
 		}
 		formatted, err := hcl.FormatMultiline(string(gen.Bytes()), loadedHCL.origin)
 		if err != nil {
-			return nil, errors.E(sm, err,
-				"failed to format generated code for block %q", name,
-			)
+			// genhcl must always generate valid code that is formatable
+			// this is a severe internal error
+			logger.Error().
```

Comment:
Coverage dropped because now we log more details + panic instead of a single return, but the condition was never tested anyway because it requires simulating a situation where terramate generates invalid HCL code, and that should never happen =P

Reply:
so maybe we can remove this `if` altogether and let it crash if it ever occurs?

Reply:
hmm the problem here is that if we remove the if it won't crash, it will just save non-formatted/broken things or an empty file (when formatting fails it returns an empty string as result, so mostly it will be an empty file).